backend/sql/initial_schema.sql

```diff
@@ -0,0 +1,100 @@
+create table project
+(
+    project_id  bigserial     not null primary key,
```

Comment:
Minor, but we actually need it to be :so-big:? Projects are created manually, so I doubt it will ever be greater than 2147483647.

Reply:
the point of having bigserial not the size but rather it's autoincremented

Reply:
I understand, but why not just `serial`?